vendored-meson/entrypoint/meson

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import re
+import sys
+import pathlib
+
+
+# The numpy-vendored version of Meson
+meson_dir = str(pathlib.Path(__file__).resolve().parent.parent / 'meson')
+sys.path.insert(0, meson_dir)
```

Comment:
Maybe assert that this directory exists?

Reply:
done now